README.md

```diff
@@ -393,6 +393,38 @@ Usage:
 ```
 
 ---
+### Logger
+#### pretty_time ([source](macros/logger/pretty_output_msg.sql))
+This macro returns a string of the current timestamp, optionally taking a datestring format.
+```sql
+{#- This will return a string like '14:50:34' -#}
+{{ dbt_utils.pretty_time() }}
+
+{#- This will return a string like '2019-05-02 14:50:34' -#}
+{{ dbt_utils.pretty_time(format='%Y-%m-%d %H:%M:%S') }}
+```
+
+#### pretty_output_msg
+This macro formats the input in a way that will print nicely to the command line when you `log` it.
+```
+{#- This will return a string like:
+"         + 11:07:31 | my pretty message"
+-#}
+
+{{ dbt_utils.pretty_output_msg("my pretty message") }}
+```
+
+It is expected that this macro will be used in conjuction with the [log](https://docs.getdbt.com/reference#log) macro.
+```
+{{ log(dbt_utils.pretty_output_msg("my pretty message"), info=True) }}
+```
+
+```
+11:07:28 | 1 of 1 START table model analytics.fct_orders........................ [RUN]
+         + 11:07:31 | my pretty message
```

Comment:
When I saw this PR, I imagined the output would instead look like:
```
11:07:31 + my pretty message
```

What's the motivation behind placing the timestamp on the right side of the divider?

Reply:
Ah, so I was largely following the pattern from [here](https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/materializations/insert_by_period_materialization.sql#L118) and in fact means to swap that out.

I like having it indented as it makes it clear that it's being called from within a model (to me at least). But happy to change it if you feel it's unintuitive.

Reply:
yeah, I'd go ahead and swap it around I think unless you feel strongly that it should work the way you've already implemented